### ISSUE TITLE
update doesn't update the ccv (composite content view) with the versions specified.

### ISSUE THREAD

**Issue by D43m0n** [RedHatSatellite/katello-cvmanager#26]
I've got the following YAML:

```
---
:settings:
  :user: admin
  :pass: changeme
  :uri: https://localhost
  :timeout: 300
  :org: 3
  :lifecycle: 9
  :keep: 3
  :promote_cvs: true
  :checkrepos: false
:ccv:
  ccv RHEL Server 7:
    cv EPEL 7: latest
    cv RHEL Server 7: latest
    cv RHSCL Server 7: latest
    cv Third Party Repositories for RHEL Server 7: latest
:publish:
  - cv EPEL 7
  - cv RHEL Server 7
  - cv RHSCL Server 7
  - cv Third Party Repositories for RHEL Server 7
:promote:
  - ccv RHEL Server 7
```

I first ran the publish command, which published all content views to the latest version. Then I ran the update command to update the composite content view so that all included content views would also be updated to the latest version:

```
# ./cvmanager update --config=cvmanager-rhel7.yaml
```
I get the following output:
```
Inspecting ccv RHEL Server 7
 Checking cv RHEL Server 7
 Checking cv RHSCL Server 7
 Checking cv EPEL 7
 Checking cv Third Party Repositories for RHEL Server 7
```
But when I check the versions of the included content views in the ccv in Satellite's WebUI, they're still at the same version as before running the update command.

**Comment by evgeni** [comment]
The YAML looks correct. Can you please run with `--verbose` from latest git master?

**Comment by D43m0n** [comment]
I pulled the latest from git master:
```
# git log | head                                                                                                                                                                                                               
commit 44d807503128cdf36ecfc5708304515c46c16d42
Merge: d05d8b6 c50ee03
Author: Evgeni Golov <evgeni@golov.de>
Date:   Sun Feb 12 10:23:46 2017 +0100

    Merge branch 'checkrepo-fixes'

commit c50ee033882fb505307f83bf08da9542f8b0d1f9
Merge: dd340bb 2324a68
Author: Evgeni Golov <evgeni@golov.de>
```
When I run with `--verbose` I don't see anything more than without that flag:
```
# ./cvmanager update  --verbose --config=cvmanager-rhel7.yaml 
Inspecting ccv RHEL Server 7 for HAN
 Checking cv RHEL Server 7
 Checking cv RHSCL Server 7
 Checking cv EPEL 7
 Checking cv Third Party Repositories for RHEL Server 7
```
And I definitely have the one with `--verbose`:
```
# grep verbose cvmanager
  :verbose     => false,
  opts.on("--verbose", "Get verbose logs from cvmanager") do
    @options[:verbose] = true
def puts_verbose(message)
  if @options[:verbose]
    puts_verbose "Past task was completed at #{task_completed_at}, which is after #{last_date}"
      puts_verbose "#{task['humanized']['output']} This past task will NOT trigger a Publish."
      puts_verbose "#{task['humanized']['output']} This past task will trigger a Publish."
  puts_verbose "#{task_completed_at} is before #{last_date}, will NOT trigger a Publish."
    puts_verbose "Checking Content View #{cv['name']}"
                  puts_verbose "Inspecing sync tasks to #10"
                  puts_verbose "Inspecing sync tasks to ##{taskreq['per_page']}"
                      puts_verbose "Found past task that matches repo id: #{tasker['input']['repository']['id']}"
              puts_verbose "  Repo: '#{repo['label']}'. Exiting from loop, reason: #{error.inspect}"
```
And when I check the code, there's no `puts_verbose` in the `update()` method :-)

**Comment by evgeni** [comment]
Ugh. Sorry. I thought @Rocco83 added some debug in there too. I'll add some debug stuff and come back to you for some testing.

Do you by any chance hang out on IRC in #satellite6? Makes discussions easier :)

**Comment by D43m0n** [comment]
Sorry, I don't hang out on IRC 😄 I've reached my maximum number of communication methods 📫 📧 ☎️ (and so on)

**Comment by evgeni** [comment]
Heh, fair enough. I'll drop you a notice when I have something we can work with.

**Comment by evgeni** [comment]
Hey @D43m0n, sorry for the long wait. I've been quite busy.
Anyways, good news.

1. I've added more verbose stuff to the update call
2. I found the issue you have. Your YAML lists names. But the code looks for labels (usually identical to names, but with underscores instead of spaces (this is related to https://github.com/RedHatSatellite/katello-cvmanager/issues/23, which I should fix, sometime). So just use labels in the YAML, and you'll be good.

**Comment by D43m0n** [comment]
I'll try labels, although I've read earlier in another issue I should use names. I'll try intermixing labels with spaces in the YAML file to see what happens 😁 

I realize it's more convenient to keep labels and names exactly the same, especially without spaces. Back in 2015 when we started with Satellite I thought using clear names with spaces and capital letters would improve readability for us humans while the Satellite code would use either labels or id's underneath.

**Comment by D43m0n** [comment]
An update:

I've used labels for `:ccv:`, and names for `:publish:` and `:promote:` and that indeed makes a working workflow for publish, update and promote of a composite content view with a YAML file. 👍 

Great work guys, this makes it possible to periodically update CCV's to all lifecycle environments!

**Comment by evgeni** [comment]
fixed in b997a78
